repo.go

```diff
@@ -74,12 +81,15 @@ func (r *Repo) Init(consistentSnapshot bool) error {
 	root.ConsistentSnapshot = consistentSnapshot
 	// Set root version to 1 for a new root.
 	root.Version = 1
-	if err = r.setTopLevelMeta("root.json", root); err != nil {
+	if err = r.setMeta("root.json", root); err != nil {
 		return err
 	}
-	if err = r.writeTargetWithExpires(t, data.DefaultExpires("targets")); err != nil {
+
+	t.Version = 1
```

Comment:
If `data.NewTargets` or `NewRoot()` set `Version` to 1, that breaks a number of existing tests. Currently, the version is incremented the first time the metadata is staged to the local store. Initializing the version to zero is a bit of a hack that gets the version incremented to 1 when the meta file is staged.

I agree this could be more intuitive, but we'd have to change code unrelated to delegations and it would be a breaking change for `data.NewTargets` and `data.NewRoot`. What do you think about creating an issue for this and handling this in a separate PR?

Reply:
Sg! https://github.com/theupdateframework/go-tuf/issues/272

Reply:
@ethan-lowman-dd  I'm attempting to solve this issue and I have set `Version` to 1 for `NewRoot()` and `NewTargets()`,.
So I'm wondering if I should remove the code lines that increment the version during staging since `Version` has a value of 1 by default now. Would appreciate your feedback on this.

Reply:
Hello @abs007, let's correspond in #272 please. This is a merged PR. Could you please share on that issue the problem you're having, ideally including code to reproduce?